app/assets/stylesheets/spotlight/_header.scss

```diff
@@ -125,49 +154,59 @@ $menu-link-background-color-hover: rgba(255,255,255,0.15);
   }
 
   &.with-image {
-    .site-title, .search-title {
+    .site-title,
+    .search-title {
       color: $white;
       text-shadow: 1px 1px 0 $black;
     }
+
     small {
       color: $white;
     }
-  }
 
-  .more-exhibits {
-    @extend .navbar-right;
-  }
+    .more-exhibits {
+      a {
+        color: $gray-lighter;
+        background: rgba(0, 0, 0, 0.4);
+        border-top-left-radius: $border-radius-base;
+        border-top-right-radius: $border-radius-base;
+      }
 
-  &.with-image .more-exhibits {
-    a {
-      color: $gray-lighter;
-      background: rgba(0, 0, 0, 0.4);
-      border-top-left-radius: $border-radius-base;
-      border-top-right-radius: $border-radius-base;
-    }
-    a:hover, a:focus, a:active {
-      background: rgba(0, 0, 0, 0.6);
-    }
+      a:hover,
+      a:focus,
+      a:active {
+        background: rgba(0, 0, 0, 0.6);
+      }
+
+      .dropdown-menu a {
+        background: none;
+        color: $gray;
 
-    .dropdown-menu a {
-      background: none;
-      color: $gray;
-      &:hover, &:focus, &:active {
-        background-color: $gray-lighter;
+        &:hover,
```

Comment:
Nesting should be no greater than 4, but was 5<br>Selector should have depth of applicability no greater than 2, but was 4